bin/rails

```diff
@@ -1,4 +1,4 @@
 #!/usr/bin/env ruby
-APP_PATH = File.expand_path('../config/application', __dir__)
+APP_PATH = File.expand_path("../config/application", __dir__)
```

Comment:
🎨 Deliberate standardization to use doublequotes, or unintentional autocorrect thrash?

Personally I prefer the pattern of using `"` only when you have some need for it like interpolation or escaped characters, and using `'` otherwise, but I've gotten some pushback on that from folks who prefer the consistency of universal `"`s.

Reply:
I prefer only to use `"`'s when interpolation is required otherwise I prefer to use `'`.

Reply:
Our C5 conventions for rubocop opt for universal double-quotes:

https://github.com/carbonfive/c5-conventions/blob/75c0d288bb98b5dd99093d0d41e7549a0b906d88/rubocop/.rubocop-common.yml#L97-L98

In this particular case, the diff was due to `bin/rails app:update`. Apparently in Rails 7 these bin files have switched to using double-quotes: https://github.com/rails/rails/commit/c13fee85ba3611143342ba253ff21ae061d1eac6